packages/access-api/src/service/upload-api-proxy.js

```diff
@@ -84,12 +84,10 @@ function getDefaultConnections(options) {
       ...(uploadApi.production && { url: uploadApi.production }),
       fetch,
     }),
-    ...(uploadApi.staging && {
-      [uploadApiEnvironments.staging.audience]: createUcantoHttpConnection({
-        ...uploadApiEnvironments.staging,
-        url: uploadApi.staging,
-        fetch,
-      }),
+    [uploadApiEnvironments.staging.audience]: createUcantoHttpConnection({
```

Comment:
would not this fail if `uploadApi.staging` is falsy ?

Reply:
I don't think it will, because in that case it will default to using values from `uploadApiEnvironments.staging` (which is always defined in this file)